manager-bundle/src/ContaoManager/ApiCommand/GetDotEnvCommand.php

```diff
@@ -50,12 +50,20 @@ protected function configure(): void
     protected function execute(InputInterface $input, OutputInterface $output): int
     {
         $path = $this->projectDir.'/.env';
+        $pathLocal = $path.'.local';
 
-        if (!file_exists($path)) {
+        if (!file_exists($path) && !file_exists($pathLocal)) {
```

Comment:
I think this change can be reverted because Symfony does not load the `.env.local` file if there is no `.env` file and so should we, too, don‘t you think?

Reply:
```suggestion
        if (!file_exists($path)) {
```